pkg/porter/stamp.go

```diff
@@ -63,7 +68,13 @@ func (p *Porter) IsBundleUpToDate(opts bundleFileOptions) (bool, error) {
 
 		// Check whether invocation images exist in host registry.
 		for _, invocationImage := range bun.InvocationImages {
-			isImageCached, err := p.Registry.IsImageCached(invocationImage.Image)
+			// if the invovationImage is built before using a random string tag,
```

Comment:
I'm having trouble following what this is trying to do. My best guess is that you are trying to account for bundles that were built from before this change?

If that's the case, I don't believe it's necessary since part of how we check if a bundle is out of date is make sure it was built with the same version of porter that is currently running. Since your change will be in a new version, that will automatically invalidate any old bundles and they will be rebuilt before publishing.

Reply:
Yes, your best guess is right on point! I ran into issue while doing local testing and turns out it's due to old invocation image built before this change. Thanks for catching it and explain how the check works